code/app/commodity/server.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"github.com/brahma-adshonor/micro_service/code/app/commodity/model"
+
+	"fmt"
+	"net/http"
+)
+
+func commodityHandler(w http.ResponseWriter, r *http.Request) {
+	type m model.Model
+}
+
+func main() {
+	http.HandleFunc("/commodity", commodityHandler)
```

Comment:
1. if use the default http server, will serve admin and metrics on the same port?
2. should gracefully stop?

Reply:
first version will not consider these features

Reply:
<img width="736" alt="image" src="https://user-images.githubusercontent.com/3197786/189060991-666cb4a7-8f6b-4e84-aa62-bfd7b699ccda.png">